### DISCUSSION TITLE
Ability to Manually Lock an Achievement (Admins Only)

### DISCUSSION THREAD

**Discussion by eldexterr** [RetroAchievements/RAWeb#776] [Ideas]
As the title said

**Comment by Keltron3030** [comment]
What's the use case?

**Comment by eldexterr** [comment]
The other day Nevermond requested a manual unlock, and BenGhazi unlocked the achievement for him by mistake because Nevermond actually didn't meet the requirements for the achievement so was wrongly awarded

**Comment by eldexterr** [comment]
This one:
https://retroachievements.org/ticketmanager.php?i=23445

And yesterday happened again with this one lol
https://retroachievements.org/ticketmanager.php?i=23715

**Comment by televandalist** [comment]
There's been a couple instances where I manually unlocked the ticket number instead of achievement number by mistake. :(

**Comment by kdecks** [comment]
This has been a planned feature for some time now, which would be available after we have a site audit log.

**Comment by Jamiras** [comment]
Duplicate of #716

**Comment by meleu** [comment]
The discussion in #481 is also relevant to this topic.
